api/geth_backend.go

```diff
@@ -476,6 +476,10 @@ func (b *GethStatusBackend) startNodeWithKey(acc multiaccounts.Account, password
 		return err
 	}
 
+	if inputNodeCfg.RuntimeLogLevel != "" {
+		b.config.RuntimeLogLevel = inputNodeCfg.RuntimeLogLevel
+	}
+
```

Comment:
Maybe we could move this to `loadNodeConfig`? 
It would make sense to make it there, as it's used not only in this function.

Reply:
> Maybe we could move this to `loadNodeConfig`? It would make sense to make it there, as it's used not only in this function.

Having a look...